fe/fe-core/src/main/java/com/starrocks/metric/MaterializedViewMetricsRegistry.java

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.metric;
+
+import com.codahale.metrics.Histogram;
+import com.codahale.metrics.MetricRegistry;
+import com.google.common.collect.Maps;
+import com.starrocks.catalog.Database;
+import com.starrocks.catalog.MaterializedView;
+import com.starrocks.catalog.MvId;
+import com.starrocks.common.ThreadPoolManager;
+import com.starrocks.server.GlobalStateMgr;
+
+import java.util.List;
+import java.util.Map;
+import java.util.TimerTask;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+public class MaterializedViewMetricsRegistry {
+
+    private final MetricRegistry metricRegistry = new MetricRegistry();
+    private final Map<MvId, MaterializedViewMetricsEntity> idToMVMetrics;
+    private final ScheduledThreadPoolExecutor timer;
+    private static final MaterializedViewMetricsRegistry INSTANCE = new MaterializedViewMetricsRegistry();
+
+    private MaterializedViewMetricsRegistry() {
+        idToMVMetrics = Maps.newHashMap();
+        // clear all metrics everyday
+        timer = ThreadPoolManager.newDaemonScheduledThreadPool(1, "MaterializedView-Metrics-Cleaner", true);
+        timer.scheduleAtFixedRate(new MaterializedViewMetricsRegistry.MetricsCleaner(), 0, 1L, TimeUnit.DAYS);
```

Comment:
Like `TableMetricsRegistry`, no need to make it configurable because we just need to collect the metrics in the day unit, and do the aggregation in the outer metrics system (eg. Prometheus).